docs/examples/driver_examples/Qcodes example with Ithaco.ipynb

```diff
@@ -4,393 +4,697 @@
+       "      <th>0.000579</th>\n",
+       "      <td>5.797739e-04</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0.000632</th>\n",
+       "      <td>6.324015e-04</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0.000685</th>\n",
+       "      <td>6.850098e-04</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0.000737</th>\n",
+       "      <td>7.376224e-04</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0.000790</th>\n",
+       "      <td>7.902317e-04</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0.000842</th>\n",
+       "      <td>8.428346e-04</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0.000895</th>\n",
+       "      <td>8.954364e-04</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0.000947</th>\n",
+       "      <td>9.480564e-04</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0.001000</th>\n",
+       "      <td>1.000660e-03</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                            curr\n",
+       "Keithley_smua_curr              \n",
+       "0.000001            9.443364e-07\n",
+       "0.000054            5.357243e-05\n",
+       "0.000106            1.062108e-04\n",
+       "0.000159            1.588324e-04\n",
+       "0.000211            2.114506e-04\n",
+       "0.000264            2.640623e-04\n",
+       "0.000316            3.166975e-04\n",
+       "0.000369            3.693167e-04\n",
+       "0.000422            4.219315e-04\n",
+       "0.000474            4.745453e-04\n",
+       "0.000527            5.271591e-04\n",
+       "0.000579            5.797739e-04\n",
+       "0.000632            6.324015e-04\n",
+       "0.000685            6.850098e-04\n",
+       "0.000737            7.376224e-04\n",
+       "0.000790            7.902317e-04\n",
+       "0.000842            8.428346e-04\n",
+       "0.000895            8.954364e-04\n",
+       "0.000947            9.480564e-04\n",
+       "0.001000            1.000660e-03"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "with Timer('Time Loop 2'):\n",
-    "    data = Loop(k1.a.volt[-5:5:1], 0).each(a1.volt, a2.volt).run(location='testsweep', overwrite=True)\n",
-    "    while data.sync():\n",
-    "        time.sleep(0.1)"
+    "dataset.get_data_as_pandas_dataframe()['curr']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because of the tuple structure of curr, we see every sweeping step (the first column) stored twice, meaning the top value of each step in the second column is the amplified voltage and the bottom one is the fed current, which is very close to the current value coming from the source in the first column. "
```

Comment:
This text does not seem to apply any more ?

Reply:
Correct :)